core/src/main/java/org/mule/routing/correlation/EventCorrelator.java

```diff
@@ -371,18 +370,9 @@ protected void handleGroupExpiry(EventGroup group) throws MessagingException
 
         if (isFailOnTimeout())
         {
-            MuleMessage messageCollection;
-            try
-            {
-                messageCollection = group.toMessageCollection();
-            }
-            catch (ObjectStoreException e)
-            {
-                throw new MessagingException(group.getMessageCollectionEvent(), e);
-            }
-            muleContext.fireNotification(new RoutingNotification(messageCollection, null,
+            MuleEvent messageCollection = group.getMessageCollectionEvent();
```

Comment:
messageCollection variable and messageCollectionEvent method sound weird to me. Are they talking about an event or a message?